src/components/blocks/blocks.css

```diff
@@ -12,6 +12,13 @@
     border-bottom-right-radius: $space;
 }
 
+[dir="rtl"] .blocks :global(.injectionDiv) {
```

Comment:
Is there a reason that this doesn't have a corresponding [dir="ltr"] version of the style?

Reply:
Same question in the two sets of changes below.

Reply:
In all the other places I use two versions I was sure that it would only be in the context of the gui - which explicitly sets `[dir="ltr"]`. I'm probably being overly cautious, but on the off chance that it could be outside the gui component, I wanted to leave the default settings so it applies if there's no `dir` attribute set. If the attribute isn't set, the value is "auto" or unknown, so it doesn't match the 'ltr' rule.